### DISCUSSION TITLE
upgrade to new version gives 404 in web browser.

### DISCUSSION THREAD

**Discussion by cm68** [thehighnibble/thehighnibble.github.io#66] [IMSAI8080esp replica]
![Screenshot_2023-05-22_22-55-37](https://github.com/thehighnibble/thehighnibble.github.io/assets/39012779/3dbe0f06-f7d3-468d-a198-527c0f03b87d)

so, what's the easiest way to start from scratch? clearly, my web server on the imsai is trashed.

**Comment by GrantMeStrength** [comment]
If you really think everything is screwed, reflashing the ESP will let you rebuild it from the ground up. The instructions are in the update files you can download from GitHub. Before you go to that bother, worth trying the ESP32 on a USB port directly and watching the status messages for clues.

**Comment by nbreeden2** [comment]
Did you read the release notes? The new version requires work on the SD card / web content otherwise you get a 404.

**Comment by nbreeden2** [comment]
Version 1.10 introduced Cromemco emulation which changed the web contents folder structure on the SD card. Anyone upgrading from 1.9 or earlier will need to follow the instructions. You can fix it manually or download the latest SD card image and write it over the SD card.

**Comment by cm68** [comment]
this fixed things.  still early in bringup, but got CP/M to start on the VGA!  let's close this.
